Dockerfile

```diff
@@ -45,7 +45,17 @@ RUN chmod +x /app/build/infracost
 # Application
 FROM alpine:3.13 as app
 # Tools needed for running diffs in CI integrations
-RUN apk --no-cache add ca-certificates openssl openssh-client curl git jq
```

Comment:
the old diff.sh script used git but not anymore, should we keep git for `scripts/get_version.sh` though?

Reply:
`scripts/get_version.sh` is only used by make which we don't include here anyway and don't think we'll need.